packages/example/src/client.ts

```diff
@@ -0,0 +1,44 @@
+// Copyright 2021-2023 Buf Technologies, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import {
+  createGrpcTransport,
+  createPromiseClient,
+} from "@bufbuild/connect-node";
+import { ElizaService } from "./gen/eliza_connectweb.js";
+import { stdin, stdout } from "process";
+import * as readline from "node:readline/promises";
+
+const rl = readline.createInterface(stdin, stdout);
+
+void (async () => {
+  const transport = createGrpcTransport({
```

Comment:
@timostamm Did you mean to use a `GrpcTransport` here?

Reply:
Yes, to show that the same handler servers multiple protocols. I'm not super happy about it because of H1 - feel free to change it.

Reply:
I moved it back to Connect just for consistency and because of the usage of HTTP/1.1.  Kind of felt confusing being there.